src/ui/pointcloud/qgspointcloudrendererpropsdialogbase.ui

```diff
@@ -233,20 +229,80 @@
-            <item row="0" column="0">
-             <widget class="QLabel" name="lblDrawOrder">
+            <item row="2" column="1" colspan="3">
+             <widget class="QgsOpacityWidget" name="mOpacityWidget" native="true">
+              <property name="focusPolicy">
+               <enum>Qt::StrongFocus</enum>
+              </property>
+             </widget>
+            </item>
+            <item row="3" column="0">
+             <widget class="QLabel" name="lblBlend">
               <property name="text">
-               <string>Draw order</string>
+               <string>Blending mode</string>
               </property>
              </widget>
             </item>
-            <item row="0" column="1" colspan="3">
-             <widget class="QComboBox" name="mDrawOrderComboBox">
-              <property name="sizePolicy">
-               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
-                <horstretch>4</horstretch>
-                <verstretch>0</verstretch>
-               </sizepolicy>
+            <item row="4" column="0" colspan="4">
+             <widget class="QGroupBox" name="mEyeDomeLightingGroupBox">
+              <property name="title">
+               <string>Eye Dome Lighting</string>
+              </property>
+              <layout class="QGridLayout" name="gridLayout_3">
+               <item row="1" column="0">
+                <widget class="QLabel" name="label_3">
+                 <property name="text">
+                  <string>Strength</string>
+                 </property>
+                </widget>
+               </item>
+               <item row="1" column="1">
+                <widget class="QSpinBox" name="mEdlStrength">
+                 <property name="minimum">
+                  <number>100</number>
+                 </property>
+                 <property name="maximum">
+                  <number>10000</number>
+                 </property>
+                 <property name="singleStep">
+                  <number>100</number>
+                 </property>
+                 <property name="value">
+                  <number>1000</number>
+                 </property>
+                </widget>
+               </item>
+               <item row="2" column="1">
+                <widget class="QSpinBox" name="mEdlDistance">
+                 <property name="minimum">
+                  <number>1</number>
+                 </property>
+                 <property name="maximum">
+                  <number>5</number>
+                 </property>
+                </widget>
+               </item>
+               <item row="2" column="0">
+                <widget class="QLabel" name="label_4">
+                 <property name="text">
+                  <string>Distance</string>
+                 </property>
+                </widget>
+               </item>
+               <item row="0" column="0" colspan="2">
+                <widget class="QCheckBox" name="mApplyEdlCheckbox">
```

Comment:
You can avoid the extra checkbox widget by making the Eye Dome groupbox checkable instead